Packs/CTIX/Integrations/CTIXv3/README.md

```diff
@@ -1380,4 +1380,316 @@ Lookup to get threat data
               "valid_from": 1652081902,
               "valid_until": null}],
  "total": 1}
-```
\ No newline at end of file
+```
+
+### domain
```

Comment:
please add documentation for `email` command too and add examples to the commands(use the -e flag for using a commad_examples file) - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#generate-docs.

Do you need any help with something?
Please don’t hesitate to reach out to me here or on Slack

Reply:
Which email command are you talking about? We don't have an email command.

Reply:
sorry my bad
so just add examples to the command you added to the README(domain, ip, file, url, ctix-get-all-notes, ctix-get-note-details, ctix-create-note, ctix-update-note, ctix-delete-note, ctix-make-request)

Reply:
Are these manually added? I was under the impression that the `demisto-sdk generate-docs` command should automatically generate the command examples. Is that not correct?